nucypher/network/nodes.py

```diff
@@ -805,15 +801,15 @@ def learn_from_teacher_node(self, eager=False, canceller=None):
         # These except clauses apply to the current_teacher itself, not the learned-about nodes.
         except NodeSeemsToBeDown as e:
             unresponsive_nodes.add(current_teacher)
-            self.log.info(
-                f"Teacher {str(current_teacher)} is perhaps down:{e}.")  # FIXME: This was printing the node bytestring. Is this really necessary?  #1712
+            raise
+            self.log.info(f"Teacher {str(current_teacher)} is perhaps down:{e}.")  # FIXME: This was printing the node bytestring. Is this really necessary?  #1712
```

Comment:
This logging is painful when it happens - can we do checksum address or `current_teacher.seed_node_metadata(as_teacher_uri=True)` or something simple like that. If needed, can we hit the `/public_information` endpoint after the fact to get the node bytes.

Reply:
✔️